### PR TITLE
role guard

### PR DESCRIPTION
**Pull request by udaydenduluri33**
EUI-3103

### CODE REVIEW

**Review by connorpgpmcelroy — APPROVED** [review]
LGTM

File: src/app/app.routes.ts [hunk 1]
```diff
@@ -15,8 +15,9 @@ export const ROUTES: Routes = [
   },
   {
     path: 'caseworker-details',
-    canActivate: [AuthGuard],
-    loadChildren: '../case-worker-ref-data/case-worker-ref-data.module#CaseWorkerRefDataModule'
+    canActivate: [AuthGuard, RoleGuard],
+    loadChildren: '../case-worker-ref-data/case-worker-ref-data.module#CaseWorkerRefDataModule',
+    data: {needsRole: ['cwd-admin'], roleMatching: 0 }
```
Comment (paul-graham) [inline src/app/app.routes.ts:20]: Should this be `roleMatching: RoleMatching.ALL`?

File: src/org-manager/org-manager.routing.ts [hunk 2]
```diff
@@ -13,16 +13,24 @@ import { UserDetailsComponent } from './containers/user-details/user-details.com
...
 
 export const ROUTES: Routes = [
   {
-    path: 'active-organisation',
-    component: ActiveOrganisationsComponent,
-    canActivate: [AuthGuard],
+    path: 'organisation',
+    component: PendingOrganisationsComponent,
+    canActivate: [AuthGuard, RoleGuard],
+    data: {needsRole: ['prd-admin'], roleMatching: 0 }
```
Comment (paul-graham) [inline src/org-manager/org-manager.routing.ts:23]: Same thing about the `roleMatching` property.

File: src/app/app.routes.ts (hunk 1 above)
Comment (udaydenduluri33) [reply src/app/app.routes.ts:20]: Thats the change I made. When you use enum in Guard the PROD build fails

File: src/org-manager/org-manager.routing.ts (hunk 2 above)
Comment (udaydenduluri33) [reply src/org-manager/org-manager.routing.ts:23]: Thats the change I made. When you use enum in Guard the PROD build fails

Comment (udaydenduluri33) [reply src/org-manager/org-manager.routing.ts:23]: This is fixed now.

File: src/app/app.routes.ts (hunk 1 above)
Comment (udaydenduluri33) [reply src/app/app.routes.ts:20]: This is fixed now

**Review by paul-graham — APPROVED** [review]

**Pull request closed without merging**